javascriptguide.xml

```diff
@@ -1,18 +1,8 @@
 <?xml version = '1.0'?>
 <?xml-stylesheet type="text/xsl" href="styleguide.xsl"?>
-<GUIDE title="Google JavaScript Style Guide">
-  <p class="revision">
-
-    Revision 2.93
```

Comment:
Why remove the revision?

I think we should keep it.

Like: GR-1.0, based on Google 2.93